packages/connect-popup/webpack/prod.webpack.config.ts

```diff
@@ -4,6 +4,9 @@ import HtmlWebpackPlugin from 'html-webpack-plugin';
 import TerserPlugin from 'terser-webpack-plugin';
 import CopyPlugin from 'copy-webpack-plugin';
 import * as URLS from '@trezor/urls';
+// todo: https://github.com/trezor/trezor-suite/issues/5305
+/* eslint-disable @typescript-eslint/prefer-ts-expect-error */
+// @ts-ignore
```

Comment:
note that we are already using this pattern here https://github.com/trezor/trezor-suite/blob/develop/packages/connect/e2e/__fixtures__/cardanoGetAddressDerivations.ts#L3